Program.cs

```diff
@@ -1,10 +1,39 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿//Number guessing game
+//Computer will think of a random number and user has to find it.
+
+var mode = GetMode ();

```

Comment:
In valid result.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/eea0b287-aa70-4360-872b-3a2991ab687f)